app/views/layouts/_footer.html.erb

```diff
@@ -19,7 +19,8 @@
           </p>
           <p class='text-muted'>
             Copyright © <%= Date.today.year %> Tidelift, Inc<br>
-            Open Source under <%= link_to 'AGPLv3', 'https://github.com/librariesio/libraries.io/blob/master/LICENSE.txt' %> license
+            Code is Open Source under <%= link_to 'AGPLv3', 'https://github.com/librariesio/libraries.io/blob/master/LICENSE.txt' %> license<br>
+            Data is available under <%= link_to 'CC-BY-SA', 'https://libraries.io/data' %> license
```

Comment:
Probably nice/specific to add "4.0" here (and on the experiments page) but otherwise this looks 👍

Reply:
Done. (The experiments page already mentioned 4.0)